packages/boiler-task-selenium/src/index.js

```diff
@@ -86,6 +94,17 @@ export default function(gulp, plugins, config) {
             logError({err, plugin: '[tunnel stop]'});
           }
 
+          try {
+            if (code) {
+              readLines(logFp, 50);
+            } else {
+              // Otherwise, remove the log file because it is USELESS TO US NOW
+              unlinkSync(logFp);
+            }
+          } catch (err) {
+            logError({err, plugin: '[tunnel log]'});
+          }
+
```

Comment:
if you want to be a real :neckbeard: you can make `readLines` take a callback and thunk it as well with `unlink`

```js
const logger = thunk(readLines)
const remove = thunk(unlink)

if (code) {
  yield logger(logFp, 50)
} else {
  yield remove()
}
```